src/main/java/wooteco/subway/path/domain/FareAdjusterByAge.java

```diff
@@ -0,0 +1,40 @@
+package wooteco.subway.path.domain;
+
+import java.util.Arrays;
+import java.util.function.BinaryOperator;
+
+public enum FareAdjusterByAge {
+    PRE_SCHOOLED(0, Constants.PRE_SCHOOL_MAX_BOUNDARY, getIntegerBinaryOperator(0)),
+    SCHOOL_AGED(Constants.PRE_SCHOOL_MAX_BOUNDARY, Constants.SCHOOL_AGED_MAX_BOUNDARY, getIntegerBinaryOperator(Constants.SCHOOL_AGED_RATE)),
+    ADOLESCENT(Constants.SCHOOL_AGED_MAX_BOUNDARY, Constants.ADOLESCENT_MAX_BOUNDARY, getIntegerBinaryOperator(Constants.ADOLESCENT_RATE));
+
+    private final int minInclusive;
```

Comment:
```suggestion
    private static final double SCHOOL_AGED_RATE = 0.5;

    private final int minInclusive;
```

`static class` 없이 이런 식으로 상수 선언 가능하지 않나요?